### PR TITLE
Mobile 3873

### PR DESCRIPTION
**Pull request by dpalou**

### CODE REVIEW

**Comment by crazyserver** [comment]
Thanks Dani for submitting your code. Please  rebase your code and add latest navigation improvements to the code.
Also I've found the activity icon is missing on assets folder.

**Comment by dpalou** [comment]
Thanks Pau, I added one new commit with the latest changes.

File: src/addons/mod/bigbluebuttonbn/components/index/index.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        <core-context-menu-item *ngIf="description" [priority]="800" [content]="'core.moduleintro' | translate"
+            (action)="expandDescription()" iconAction="fas-arrow-right">
+        </core-context-menu-item>
+        <core-context-menu-item *ngIf="blog" [priority]="750" content="{{'addon.blog.blog' | translate}}" iconAction="far-newspaper"
+            (action)="gotoBlog()">
+        </core-context-menu-item>
+        <core-context-menu-item *ngIf="loaded && isOnline" [priority]="700" [content]="'core.refresh' | translate"
+            (action)="doRefresh(null, $event)" [iconAction]="refreshIcon" [closeOnClick]="false">
+        </core-context-menu-item>
+        <core-context-menu-item *ngIf="loaded && hasOffline && isOnline" [priority]="600"
```
Comment (crazyserver) [inline src/addons/mod/bigbluebuttonbn/components/index/index.html:16]: is hasOffline defined and being used?

File: src/addons/mod/bigbluebuttonbn/components/index/index.html [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        </core-context-menu-item>
+    </core-context-menu>
+</core-navbar-buttons>
+
+<!-- Content. -->
+<core-loading [hideUntil]="loaded" class="safe-area-page">
+
+    <!-- Activity info. -->
+    <core-course-module-info [module]="module" (completionChanged)="onCompletionChange()" [description]="description"
+        [component]="component" [componentId]="componentId" [courseId]="courseId" [hasDataToSync]="hasOffline">
```
Comment (crazyserver) [inline src/addons/mod/bigbluebuttonbn/components/index/index.html:31]: is hasOffline defined and being used?

File: src/addons/mod/bigbluebuttonbn/components/index/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+<!-- Content. -->
+<core-loading [hideUntil]="loaded" class="safe-area-page">
+
+    <!-- Activity info. -->
+    <core-course-module-info *ngIf="showCompletion" [module]="module" [showManualCompletion]="true"
+        (completionChanged)="onCompletionChange()">
+    </core-course-module-info>
+
+    <core-course-module-description [description]="description" [component]="component" [componentId]="componentId"
```
[the remark this answers is not in the document]
Comment (dpalou) [reply src/addons/mod/bigbluebuttonbn/components/index/index.html]: Done.

File: src/addons/mod/bigbluebuttonbn/pages/index/index.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<ion-header>
+    <ion-toolbar>
+        <ion-buttons slot="start">
+            <ion-back-button [text]="'core.back' | translate"></ion-back-button>
+        </ion-buttons>
+        <h1>
```
[the remark this answers is not in the document]
Comment (dpalou) [reply src/addons/mod/bigbluebuttonbn/pages/index/index.html]: Done.

File: src/addons/mod/bigbluebuttonbn/components/index/index.ts [hunk 5]
```diff
@@ -45,6 +45,7 @@ export class AddonModBBBIndexComponent extends CoreCourseModuleMainActivityCompo
         @Optional() courseContentsPage?: CoreCourseContentsPage,
     ) {
         super('AddonModBBBIndexComponent', content, courseContentsPage);
+        (<any>window).ths = this;
```
[the remark this answers is not in the document]
Comment (dpalou) [reply src/addons/mod/bigbluebuttonbn/components/index/index.ts]: Yes it was lol. It's not the first time I push one of these ^^

File: src/addons/mod/bigbluebuttonbn/components/index/index.html [hunk 6]
```diff
@@ -35,6 +35,96 @@
...
+            {{ 'addon.mod_bigbluebuttonbn.view_conference_action_join' | translate }}
+        </ion-button>
+
+        <ion-card *ngIf="!meetingInfo.canjoin" class="core-warning-card">
+            <ion-item class="ion-text-wrap">
+                <ion-icon name="fas-exclamation-triangle" slot="start" aria-hidden="true"></ion-icon>
+                <ion-label>{{ meetingInfo.statusmessage }}</ion-label>
+            </ion-item>
+        </ion-card>
+    </ng-container>
```
[the remark this answers is not in the document]
Comment (dpalou) [reply src/addons/mod/bigbluebuttonbn/components/index/index.html:126]: Done.

File: src/addons/mod/bigbluebuttonbn/components/index/index.html (hunk 1 above)
Comment (dpalou) [reply src/addons/mod/bigbluebuttonbn/components/index/index.html]: Nope, I removed it, nice catch.

File: src/addons/mod/bigbluebuttonbn/components/index/index.ts (hunk 5 above)
[the remark this answers is not in the document]
Comment (crazyserver) [reply src/addons/mod/bigbluebuttonbn/components/index/index.ts]: PRobably we should add some check like the console on on our CI scripts?

**Review by crazyserver — APPROVED** [review]

**Pull request merged**